rdr_service/dao/bq_participant_summary_dao.py

```diff
@@ -636,12 +636,13 @@ def _prep_the_basics(self, p_id, ro_session):
         :param ro_session: Readonly DAO session object
         :return: dict
         """
-        qnans = self.ro_dao.call_proc('sp_get_questionnaire_answers', args=['TheBasics', p_id])
+        qnans = self.get_module_answers(self.ro_dao, 'TheBasics', p_id, return_responses=False,
+                                        cdm_db_exists=self.cdm_db_exists)
         if not qnans or len(qnans) == 0:
             return {}
 
         # get race question answers
-        qnan = BQRecord(schema=None, data=qnans[0])  # use only most recent questionnaire.
+        qnan = BQRecord(schema=None, data=qnans)  # use only most recent questionnaire.
```

Comment:
Comment may no longer apply?

Reply:
The old 'sp_get_questionnaire_answers' stored proc returned multiple rows, with the most recent response first.  The 'get_module_answers()' class method only returns a single dict object with the most recent response answers.

Reply:
I changed the comment.

Reply:
My bad.  I was referring to "use only most recent questionnaire"